charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -125,6 +125,9 @@ spec:
                   mountPath: /var/lib/clickhouse
               resources:
 {{ toYaml .Values.clickhouse.resources | indent 16 }}
+              {{- if .Values.clickhouse.nodeSelector }}
+              nodeSelector: {{- .Values.clickhouse.nodeSelector | toYaml | nindent 16 }}
```

Comment:
this may not have the correct indent, I think this should be at the same indent as the tolerations?

Reply:
Looking at [this example](https://github.com/Altinity/clickhouse-operator/issues/450#issuecomment-665728125) I think it should be in the `spec` tree?

Reply:
yes it should be part of the pod spec template and not the container

Reply:
You are right, the PR is using a bad indentation. I need a pair of 👓